pennylane/transforms/decompose.py

```diff
@@ -203,7 +204,7 @@ def _evaluate_jaxpr_decomposition(self, op: Operator):
             if self.max_expansion is not None and self._current_depth >= self.max_expansion:
                 return self.interpret_operation(op)
 
-            if enabled_graph() and self._decomp_graph_solution.is_solved_for(op):
+            if self._decomp_graph_solution and self._decomp_graph_solution.is_solved_for(op):
```

Comment:
Is this change needed for this specific PR or is it just low hanging fruit? I don't mind either way, just curious.